### PR TITLE
Add Ougai for logging

### LINKED ISSUE #2913: try using https://github.com/tilfin/ougai
https://github.com/tilfin/ougai

lets do better logging!

### Questions? Join Slack!

We highly recommend that you join us in slack https://rubyforgood.herokuapp.com/ #casa channel to ask questions quickly and hear about office hours (currently Wednesday 6-8pm Pacific), stakeholder news, and upcoming new issues.

### PR DESCRIPTION
**Pull request by ludamillion**
### What github issue is this PR for, if any?
Resolves #2913

### What changed, and why?
Added and configured the Ougai gem for logging

### How is this tested? (please write tests!) 💖💪
Manually. We don't seem to have any specific logger calls that can/should be tested.

### CODE REVIEW

**Review by littleforest — COMMENTED** [review]
Thanks for setting this up! Do we need to install bunyan per the [docs](https://github.com/tilfin/ougai#view-pretty-logs-with-node-bunyan-or-pino)?

**Comment by ludamillion** [comment]
My impression was that part of the docs was for viewing the logs locally. I don’t think it involved in the logging process itself. But I’ll look into to make sure.

**Comment by ludamillion** [comment]
As I thought bunyan is really only required for viewing logs but I went ahead and added it as a dev dependency.

File: package.json [hunk 1]
```diff
@@ -36,6 +36,7 @@
   "version": "0.1.0",
   "devDependencies": {
     "@babel/preset-env": "^7.19.3",
+    "bunyan": "^1.8.15",
```
Comment (compwron) [inline package.json:39]: https://www.npmjs.com/package/bunyan

**Review by compwron — APPROVED** [review]
let's try it! :)

**Pull request merged**